### PR TITLE
Mozilla support and css updates

### PR DESCRIPTION
**Pull request by Xesme**
This branch adds updates to styling and enables Mozilla Firefox support.  Issue #81

### CODE REVIEW

File: app/src/scripts/common/partials/happybrowser.jsx [hunk 1]
```diff
@@ -56,12 +57,16 @@ export default class Happybrowser extends React.Component {
...
   _incompatibleBrowser(ua) {
     const check = bowser.check(
-      { chrome: '49', chromium: '49', googlebot: '0' },
+      { chrome: '49', chromium: '49', googlebot: '0', firefox: '50' },
       true,
       ua || window.navigator.userAgent,
     )
     if (
-      bowser.check({ chrome: '41' }, true, ua || window.navigator.userAgent)
+      bowser.check(
```
Comment (nellh) [inline app/src/scripts/common/partials/happybrowser.jsx]: This check should not include Firefox, it is a fallback for Googlebot (Chrome 41 only).

File: app/src/scripts/common/partials/happybrowser.jsx [hunk 2]
```diff
@@ -31,7 +31,8 @@ export default class Happybrowser extends React.Component {
               site may not work as expected.{' '}
               <strong>
                 <a href="http://www.google.com/chrome/">
-                  Please consider using Chrome, V49.0 or higher, as your browser
+                  Please consider using Chrome or Firefox, V49.0 or higher, as
```
Comment (nellh) [inline app/src/scripts/common/partials/happybrowser.jsx]: This should specify Chrome 49 and Firefox 50.

File: app/src/sass/crn_app/_elements-wrappers.scss [hunk 3]
```diff
@@ -78,6 +78,7 @@ body {
 
 a {
     color: $primary;
+    font-weight: 600;
```
Comment (nellh) [inline app/src/sass/crn_app/_elements-wrappers.scss]: This seems to change the font-weight for all of the analysis views. Is that intentional?

**Review by nellh — CHANGES_REQUESTED** [review]
Have a few screenshots of other issues I'll add to the PR comments.

**Comment by nellh** [comment]
Datasets dashboard filters wrap in Firefox.

datasets-wrap

This "draft dataset has been deleted" message is visible in Firefox but not Chrome.

draft-dataset

Panels in the analysis components have some extra space.

job-status-spacing

In both Firefox and Chrome, the upload dataset tabs are no longer aligned.

tabs

**Comment by nellh** [comment]
I've also noticed two upload bugs with Firefox.

* The check for a dataset with the same name does not work, Firefox will let you upload a dataset with the same name as one uploaded by Chrome and silently fail out of the upload process.
* Somehow "%3A" got prepended to dataset_descripton.json's filename on a successful upload that had been resumed. "%3Adataset_description.json".

**Comment by Xesme** [comment]
@nellh I think we may be having some os/linux styling issues. 

I can see 'This "draft dataset has been deleted" message is visible in Firefox but not Chrome.' within both FF and Chrome.
screen shot 2017-11-07 at 2 17 33 pm

Also, my datasets dashboard filters do not wrap in FF.

screen shot 2017-11-07 at 2 17 49 pm
screen shot 2017-11-07 at 2 18 05 pm

**Comment by nellh** [comment]
@Xesme Is there a good reason to not remove the width on `.filters-sort-wrap .filters` - it seems this fixes the Linux wrapping issue with the dataset filters.

**Comment by Xesme** [comment]
@nellh unfortunately, it was added as a fix to wrapping filters on OS. I will try to find another work around.

**Comment by Xesme** [comment]
@nell originally I had added it to prevent wrapping on OS Mozilla, I just tested it by taking it it out, seems like it's not causing an issue anymore. I think it may be related to the same issue within the upload modal, which was an inherited style that has been changed. I will push a fix right now.

**Comment by Xesme** [comment]
@nell originally I had added it to prevent wrapping on OS Mozilla, I just tested it by taking it it out, seems like it's not causing an issue anymore. I think it may be related to the same issue within the upload modal, which was an inherited style that has been changed. I will push a fix right now.

**Comment by Xesme** [comment]
@nell originally I had added it to prevent wrapping on OS Mozilla, I just tested it by taking it it out, seems like it's not causing an issue anymore. I think it may be related to the same issue within the upload modal, which was an inherited style that has been changed. I will push a fix right now.

**Review by nellh — APPROVED** [review]

**Pull request merged**